backend/tests/test_api/test_evidence.py

```
@@ -0,0 +1,178 @@
+"""
+
+import pytest
+from fastapi import status
+
+
+class TestEvidencePresignedUrl:
+    """
+    Test suite for POST /evidence/presigned-url endpoint
+    """
+
+    def test_should_generate_presigned_url(self, client, test_user, auth_headers):
```

Ruff (0.14.5)

Unused method argument: `test_user`

(ARG002)